src/components/App.css

```diff
@@ -46,8 +53,8 @@
   100%  {opacity: 1;}
 }
 
-@media (min-width: 320px) and (max-width: 480px) {
-  
+@media (max-width: 480px) {
```

Comment:
if screen grew shorter than 320 px, the layout will return to that of big screens. So just kept the 480px mark